tests/controls/test_control_decorators.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from IPython.display import HTML
+
+import seismometer.controls.decorators as under_test
+
+
+class BadReverse:
+    def __init__(self, string):
+        self.string = string
+        self.reverse = string[::-1]
+
+    def __str__(self):
+        return self.string
+
+    def __repr__(self):
+        return f"BadReverse({self.string})"
+
+
+class Test_Cached_Function:
+    def teardown_method(self):
```

Comment:
This feels more unittest-y   
Is there something that fits better in those patterns than creating a fixture that manipulates state, yields, and performs clean up?  The fixture could include your enabling line too.

Reply:
Yeah, using a fixture does make more sense